_posts/en/newsletters/2021-07-14-newsletter.md

```diff
@@ -0,0 +1,223 @@
+    could check almost any part of the transaction that later spends
+    those bitcoins.  This can enable many advanced features (including
+    versions[^expensive] of other proposed upgrades like
+    [SIGHASH_ANYPREVOUT][topic sighash_noinput] and
+    [OP_CHECKTEMPLATEVERIFY][topic op_checktemplateverify]), but
+    `OP_CAT` also makes it possible to create recursive
+    [covenants][topic covenants] which could permanently restrict the
+    spendability of any bitcoins committed to the covenant.  Some people
+    have objected to allowing covenants in Bitcoin, but several
+    [arguments][harding altcoins] were [made][towns multisig] to the
+    effect that the worst case problems of recursive covenants already
+    exist in Bitcoin today so we shouldn't be concerned about enabling
+    `OP_CAT` or a similar opcode.
+
+    Despite the discussion, Rubin decided he wanted to keep his
+    `OP_CSFS` proposal independent of any proposal to add `OP_CAT`,
+    [arguing][rubin just csfs] that `OP_CSFS` is useful enough on its
+    own.
+
+- **Tracking bech32m support:** the Bitcoin Wiki page for [bech32
+  adoption][wiki bech32 adoption] has been [updated][erhardt bech32m
+  tweet] to track which software and services support spending or
+  receiving to [bech32m][topic bech32] addresses for taproot.
+
+## Bitcoin Core PR Review Club
+
+*In this monthly section, we summarize a recent [Bitcoin Core PR Review Club][]
+meeting, highlighting some of the important questions and answers.  Click on a
+question below to see a summary of the answer from the meeting.*
+
+[Use script_util helpers for creating P2{PKH,SH,WPKH,WSH} scripts][review club #22363] is a PR by
+Sebastian Falbesoner which substitutes manual script creation with calls to `script_util` helper
+functions in functional tests and fixes an error in the `get_multisig()` function. The review club
+meeting broke down terminology and each of the script output types used in the PR.
+
+{% include functions/details-list.md
+
+  q0="What do `key_to_p2pkh_script`, `script_to_p2sh_script`, `key_to_p2wpkh_script` and
+`script_to_p2wsh_script` in script\_util.py do?" a0="These are helper functions to construct
+`CScript` objects for Pay to Public Key Hash, Pay to Script Hash, Pay to Witness Public Key Hash,
+and Pay to Witness Script Hash scripts from public keys and scripts."
+  a0link="https://bitcoincore.reviews/22363#l-17"
+
+  q1="Define scriptPubKey, scriptSig and witness."
+  a1="The scriptSig and scriptPubKey are fields in the input and output of a transaction,
+respectively, for specifying spending conditions. The witness is an additional field for the same
+purpose introduced with Segregated Witness. Spending requirements are committed to in an output's
+scriptPubKey and the input that spends it must be accompanied by a satisfying script in the
+scriptSig and/or witness."
+  a1link="https://bitcoincore.reviews/22363#l-31"
+
+  q2="Define witness program and scriptCode."
```

Comment:
I think `scriptCode` is quite confusing and is used to mean more than one thing. I suggest you remove this question/answer.